app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -417,4 +422,22 @@ <h2>Copy</h2>
 {% block script %}
     {{ block.super }}
     <script src="{% static "js/refresh_sidebar.js" %}"></script>
+    <script>
+        window.drf = {
+            csrfHeaderName: "{{ csrf_header_name|default:'X-CSRFToken' }}",
+            csrfToken: "{% if request %}{{ csrf_token }}{% endif %}"
+        };
+        deleteGroundTruth = () => {
+            $.ajax({
+                url: "{% url 'api:reader-study-remove-ground-truth' pk=object.pk %}",
+                type: "POST",
+                complete: () => {location.reload()},
+                headers: {
+                    'X-CSRFToken': window.drf.csrftoken,
+                    'Content-Type': 'application/json'
+                },
+            });
+        };
```

Comment:
Please move this to its own js file, we have to get rid of all inline js so that we can set a proper CSP (TODO for a long time).